packages/selfie-segmentation/android/src/main/java/io/capawesome/capacitorjs/plugins/mlkit/selfiesegmentation/SelfieSegmentationPlugin.java

```diff
@@ -1,22 +1,81 @@
 package io.capawesome.capacitorjs.plugins.mlkit.selfiesegmentation;
 
-import com.getcapacitor.JSObject;
+import com.getcapacitor.Logger;
 import com.getcapacitor.Plugin;
 import com.getcapacitor.PluginCall;
 import com.getcapacitor.PluginMethod;
 import com.getcapacitor.annotation.CapacitorPlugin;
+import com.google.mlkit.vision.common.InputImage;
+import io.capawesome.capacitorjs.plugins.mlkit.selfiesegmentation.classes.ProcessImageOptions;
+import io.capawesome.capacitorjs.plugins.mlkit.selfiesegmentation.classes.ProcessImageResult;
 
 @CapacitorPlugin(name = "SelfieSegmentation")
 public class SelfieSegmentationPlugin extends Plugin {
 
-    private SelfieSegmentation implementation = new SelfieSegmentation();
+    public static final String TAG = "SelfieSegmentation";
+    public static final String ERROR_PROCESS_IMAGE_CANCELED = "processImage canceled.";
+    public static final String ERROR_PATH_MISSING = "path must be provided.";
+    public static final String ERROR_LOAD_IMAGE_FAILED = "image could not be loaded.";
+
+    private SelfieSegmentation implementation;
+
+    @Override
+    public void load() {
+        try {
+            implementation = new SelfieSegmentation(this);
+        } catch (Exception exception) {
+            Logger.error(TAG, exception.getMessage(), exception);
+        }
+    }
 
     @PluginMethod
-    public void echo(PluginCall call) {
-        String value = call.getString("value");
+    public void processImage(PluginCall call) {
+        try {
+            String path = call.getString("path", null);
+            if (path == null) {
+                call.reject(ERROR_PATH_MISSING);
+                return;
+            }
+            Boolean enableRawSizeMask = call.getBoolean("enableRawSizeMask", false);
```

Comment:
Please add the `@default false` jsdoc tag to the `enableRawSizeMask` property in `definitions.ts.`